dll/win32/netcfgx/lang/ro-RO.rc

```diff
@@ -1,46 +1,52 @@
-// COPYRIGHT: Ștefan Fulea <stefan.fulea@mail.com>
+/*
+ * PROJECT:     ReactOS Configuration of network devices
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Romanian resource file
+ * TRANSLATORS: Copyright 2011-2018 Ștefan Fulea <stefan.fulea@mail.com>
+ *              Copyright 2024 Andrei Miloiu <miloiuandrei@gmail.com>
+ */
 
 LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
 
 IDD_TCPIP_BASIC_DLG DIALOGEX 0, 0, 246, 216
 STYLE DS_SHELLFONT | WS_CHILD | WS_CAPTION
-CAPTION "Generale"
+CAPTION "General"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Puteți obține automat configurarea adresei IP în rețelele cu DHCP. Altfel, este necesară configurarea manuală de către administratorul de rețea.", -1, 9, 9, 235, 27
+    LTEXT "Dacă reţeaua acceptă, aveţi posibilitatea să obţineţi setări IP atribuite automat. În caz contrar, este necesar să cereţi setările potrivite de la administratorului reţelei.", -1, 9, 9, 235, 27
     GROUPBOX "", -1, 9, 61, 228, 60, BS_GROUPBOX
-    CONTROL "&Obține automat adresă IP", IDC_USEDHCP, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 14, 43, 210, 12
-    CONTROL "Folosește adresa ur&mătoare:", IDC_NODHCP, "BUTTON", BS_AUTORADIOBUTTON, 14, 59, 120, 12
+    CONTROL "Se &obţine automat o adresă IP", IDC_USEDHCP, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 14, 43, 210, 12
+    CONTROL "Se &utilizează adresa IP următoare::", IDC_NODHCP, "BUTTON", BS_AUTORADIOBUTTON, 14, 59, 120, 12
     LTEXT "Adresă I&P:", -1, 14, 75, 135, 8
     CONTROL "", IDC_IPADDR, "SysIPAddress32", WS_TABSTOP, 144, 75 - 2, 85, 12
-    LTEXT "Mas&că de subrețea:", -1, 14, 90, 135, 8
+    LTEXT "&Mască subreţea:", -1, 14, 90, 135, 8
     CONTROL "", IDC_SUBNETMASK, "SysIPAddress32", WS_TABSTOP, 144, 90 - 2, 85, 12
-    LTEXT "S&erver „poartă” implicit:", -1, 14, 105, 135, 8
+    LTEXT "&Gateway implicit:", -1, 14, 105, 135, 8
     CONTROL "", IDC_DEFGATEWAY, "SysIPAddress32", WS_TABSTOP, 144, 105 - 2, 85, 12
     GROUPBOX "", -1, 9, 147, 228, 47, BS_GROUPBOX
-    CONTROL "Obține a&utomat adresa serverului DNS", IDC_AUTODNS, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 14, 129, 210, 12
-    CONTROL "Folosește adresa &DNS următoare", IDC_FIXEDDNS, "BUTTON", BS_AUTORADIOBUTTON, 14, 145, 152, 12
-    LTEXT "Server DNS p&referat:", -1, 14, 161, 135, 8
+    CONTROL "Se o&bţine automat adresa de server DNS", IDC_AUTODNS, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 14, 129, 210, 12
+    CONTROL "S&e utilizează adresa de server DNS următoare:", IDC_FIXEDDNS, "BUTTON", BS_AUTORADIOBUTTON, 14, 145, 152, 12
```

Comment:
Are you sure this ends with a `:` ?

Reply:
Yes, I'm sure. Here is the proof.

![2024-05-09 22_39_28-](https://github.com/reactos/reactos/assets/113553683/566e5146-296f-4096-8d28-29b7aaa59058)